<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="FCU-NBK500" failures="0" tests="0" name="Test2" time="3.093" errors="0" timestamp="2021-01-22T23:26:25 CET">
  <testcase classname="de.alain.FreelancerDe.setup.SetUp" name="@AfterTest EndTest" time="3.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at de.alain.FreelancerDe.setup.SetUp.EndTest(SetUp.java:121)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest EndTest -->
</testsuite> <!-- Test2 -->
